source/api_documentation/network_integration/js_tags/index.rst

```diff
@@ -0,0 +1,182 @@
+Invoca Tags
+============
+
+Manage Invoca Tags (aka JS Tags)
+""""""""""""""""""""""""""""""""""""""""""""""""
+
+The Invoca Tag is a snippet of code that connects Invoca to your landing pages. This API documentation helps provide necessary features to create and manage an Invoca Tag and it's revisions.
+
+.. raw:: html
+
+    <p>
+    More details on the Invoca Tags can be found <a href="https://community.invoca.com/t5/call-tracking/how-to-configure-and-deploy-your-invoca-tag-with-the-invoca-tag/ta-p/465#U465">here</a> and <a href="https://community.invoca.com/t5/developer-features/invoca-tag-wizard-custom-code-settings/ta-p/691">here</a>.
```

Comment:
For the links I think it would be good to format it a little differently, and show the title of the articles. I'm imagining a bulleted list with the links

Reply:
Done in bc7334279036ff8e8f04d57911e997d29e08bfc8

<img width="1409" alt="Screenshot 2023-11-09 at 12 11 37 AM" src="https://github.com/Invoca/developer-docs/assets/1645158/7f0d0e85-9fc3-41b3-8b46-3cb2d8242bf7">